rotkehlchen/tests/api/test_pnl_csv.py

```diff
@@ -157,6 +162,34 @@ def test_history_export_download_csv(
         with zipfile.ZipFile(tempzipfile, 'r') as zip_ref:
             zip_ref.extractall(extractdir)
         assert_csv_export_response(response, extractdir, is_download=True)
+    # check if it works with custom assets
```

Comment:
I would say leave a space above this line

```suggestion
    # check if the report works when using assets that don't have a symbol
```